package.json

```diff
@@ -65,14 +67,14 @@
     "rollup-plugin-scss": "~2.1.0",
     "rollup-plugin-sourcemaps": "~0.5.0",
     "rollup-plugin-terser": "~6.1.0",
-    "rollup-plugin-typescript2": "~0.25.3",
+    "rollup-plugin-typescript2": "~0.30.0",
     "rollup-plugin-visualizer": "^4.0.4",
     "rollup-plugin-vue": "~5.1.5",
     "sass-loader": "~8.0.0",
-    "typescript": "~3.9.6",
+    "source-map-loader": "~1.1.3",
```

Comment:
Why did you add this package?

Reply:
To see the source maps when we debug in Browser and we serve the project locally.